feature/policy_forwarding/decapsulation/otg_tests/ip_guev1_static_decap_subnet_range/ip_guev1_static_decap_subnet_range_test.go

```diff
@@ -0,0 +1,642 @@
+		} else {
+			t.Errorf("Fail : policer counters not incremented after start and stop traffic")
+		}
+	} else {
+		t.Logf("Traffic Packet Counters on DUT based on Policer. Expecting no packet Increment after Traffic ")
+		if finalpacket-intialpacket == 0 {
+			t.Logf("Pass : policer counters did not incremented as expected")
+		} else {
+			t.Errorf("Fail : policer counters incremented unexpectedly")
+		}
+	}
+}
+
+func gueDecapInnerIpv4Traffic(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, topo gosnappi.Config, ateUdpPort int, dutUdpPort int, destIp string, trafficValidation bool, verifyCounters bool, countersMatch bool) {
+	trafficID := fmt.Sprintf("Gue-Decap-Flow1-%v", ateUdpPort)
+	configureIPv4Traffic(t, ate, topo, trafficID, ateUdpPort, destIp)
+	configureDutWithGueDecap(t, dut, dutUdpPort, "ipv4")
+	protocolStart(t, dut, ate, topo)
+	startCapture(t, ate)
+	intialpacket1 := verfiy_policer_matched_packets(t, dut)
+	trafficStartStop(t, ate, topo)
+	stopCapture(t, ate)
+	if trafficValidation {
+		verifyTrafficFlow(t, ate, trafficID, true)
+		verifyCaptureDscpTtlValue(t, ate, "port2", 32, 49)
+	} else {
+		verifyTrafficFlow(t, ate, trafficID, false)
+	}
+	finalpacket1 := verfiy_policer_matched_packets(t, dut)
+	if verifyCounters {
+		compare_counters(t, intialpacket1, finalpacket1, countersMatch)
+	}
+}
+
+func gueDecapInnerIpv6Traffic(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, topo gosnappi.Config, ateUdpPort int, dutUdpPort int, destIp string, trafficValidation bool, verifyCounters bool, countersMatch bool) {
+	trafficID := fmt.Sprintf("Gue-Decap-Flow1-%v", ateUdpPort)
+	configureIPv6Traffic(t, ate, topo, trafficID, ateUdpPort, destIp)
+	configureDutWithGueDecap(t, dut, dutUdpPort, "ipv6")
+	protocolStart(t, dut, ate, topo)
+	intialpacket1 := verfiy_policer_matched_packets(t, dut)
+	startCapture(t, ate)
+	trafficStartStop(t, ate, topo)
+	stopCapture(t, ate)
+	if trafficValidation {
+		verifyTrafficFlow(t, ate, trafficID, true)
+		verifyCaptureDscpTtlValue(t, ate, "port2", 32, 49)
+	} else {
+		verifyTrafficFlow(t, ate, trafficID, false)
+	}
+	finalpacket1 := verfiy_policer_matched_packets(t, dut)
+	if verifyCounters {
+		compare_counters(t, intialpacket1, finalpacket1, countersMatch)
+	}
+}
+
+func configureDutWithGueDecap(t *testing.T, dut *ondatra.DUTDevice, guePort int, ipType string) {
+	t.Logf("Configure DUT with decapsulation UDP port %v", guePort)
+	ocPFParams := GetDefaultOcPolicyForwardingParams(t, dut, guePort, ipType)
+	_, _, pf := cfgplugins.SetupPolicyForwardingInfraOC(ocPFParams.NetworkInstanceName)
+	cfgplugins.DecapGroupConfigGue(t, dut, pf, ocPFParams)
+}
+
+// GetDefaultOcPolicyForwardingParams provides default parameters for the generator,
+// matching the values in the provided JSON example.
+func GetDefaultOcPolicyForwardingParams(t *testing.T, dut *ondatra.DUTDevice, guePort int, ipType string) cfgplugins.OcPolicyForwardingParams {
```

Comment:
Since this function is not exported, it should be lowercase 

```suggestion
func getDefaultOcPolicyForwardingParams(t *testing.T, dut *ondatra.DUTDevice, guePort int, ipType string) cfgplugins.OcPolicyForwardingParams {
```